dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/IndiciesInfo.java

```diff
@@ -0,0 +1,234 @@
+    }
+
+    /**
+     * Return the timestamp from a index name, a index name has the follow sintax:
+     *
+     * <b>cluster_<CLUSTER_ID>.<INDEX_NAME_PREFIX>_<TIME_STAMP></b>
+     * @return
+     */
+    public long getIndexTimeStamp(final IndexType indexType) {
+        Date startTime;
+        try {
+            final String indexName = indiciesNames.get(indexType);
+            final String indexTimestamp = indexName.substring(indexName.lastIndexOf("_") + 1);
+            startTime = timestampFormatter.parse(indexTimestamp);
+
+            return System.currentTimeMillis() - startTime.getTime();
+        } catch (ParseException e) {
+            throw new DotRuntimeException(e);
+        }
+    }
+
+    /**
+     * Create a new indicies name for the type pass in indiciesType
+     *
+     * @param indiciesType types for what you want to create a new name
+     * @return timestamp for the newly indicies, to get the newly indices name use one of the follow methods:
+     * {@link IndiciesInfo#getLive()}, {@link IndiciesInfo#getWorking()}, {@link IndiciesInfo#getReindexLive()},
+     * {@link IndiciesInfo#getReindexWorking()} {@link IndiciesInfo#getSiteSearch()}
+     */
+    public String createNewIndiciesName(final IndexType... indiciesType) {
+        final String timeStamp = timestampFormatter.format(new Date());
+
+        for (final IndexType indexType : indiciesType) {
+            final String indexName = String.format(
+                    INDEX_NAME_PATTERN,
+                    ClusterFactory.getClusterId(),
+                    indexType.getPrefix(),
+                    timeStamp);
+
+            this.indiciesNames.put(indexType, indexName);
+        }
+
+        return timeStamp;
+    }
+
+    public Map<IndexType, String> asMap() {
+        final Map<IndexType, String> actives = new HashMap<>();
+        for (IndexType type : IndexType.values()) {
+            final String newValue = Try.of(() -> (String) PropertyUtils
+                    .getProperty(this, type.getPropertyName())).getOrNull();
+            if (newValue != null) {
+                actives.put(type, newValue);
+            }
+
+        }
+        return actives;
+    }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + ((this.getLive() == null) ? 0 : this.getLive().hashCode());
+        result = prime * result + ((this.getReindexLive() == null) ? 0 : this.getReindexLive().hashCode());
+        result = prime * result + ((this.getReindexWorking() == null) ? 0 : this.getReindexWorking().hashCode());
+        result = prime * result + ((this.getSiteSearch() == null) ? 0 : this.getSiteSearch().hashCode());
+        result = prime * result + ((this.getWorking() == null) ? 0 : this.getWorking().hashCode());
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
+            return true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This statement should have braces](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)